low-level/files/opt/seagate/sspl/setup/sspl_config.py

```diff
@@ -30,13 +30,13 @@
 import errno
 import consul
 
-from cortx.sspl.bin import sspl_constants as consts
+from framework.base import sspl_constants as consts
 from cortx.utils.service import Service
 from cortx.utils.process import SimpleProcess
 from cortx.utils.validator.v_service import ServiceV
-from cortx.sspl.lowlevel.files.opt.seagate.sspl.setup.error import SetupError
+from error import SetupError
```

Comment:
this is in site-pkgs/cortx/sspl ? I thought it was in sspl/bin. We need to rename this 'error.py' also to setup_error.py to reflect whats in it

Reply:
Now we no need to create soft link n site-packages.
Done